### DISCUSSION TITLE
Summary report of multiple projects

### DISCUSSION THREAD

**Discussion by Pl8tinium** [DependencyTrack/dependency-track#2425] [General]
Hey 

i am searching for a way to summarize/ group multiple project reports into one to have a better overview over a project that consists of multiple applications. 

my ideas:

- i was wondering if it makes sense to merge related sboms into one to have them displayed in one project (probably not the best idea)
- to pull out the data via the api and visualize it myself (wanna avoid that effort at best)

- use an existing tool that can parse multiple outputs into one

Did you already have similar thoughts or maybe there are even some solutions i just dont know about yet?

**Comment by nscuro** [comment]
I think this describes what you're asking for: https://github.com/DependencyTrack/dependency-track/issues/2041

**Reply by Pl8tinium** [reply]
exactly thank you, i guess ill just create some summarization reports by myself
